src/homepage2/HomePage.js

```diff
@@ -5,7 +5,23 @@ import React from "react";
 import type {Assets} from "../webutil/assets";
 import {StyleSheet, css} from "aphrodite/no-important";
 
+async function loadAndReport(assets, path) {
+  const url = assets.resolve(path);
+  const response = await fetch(url);
+  if (!response.ok) {
+    console.error(path, response);
+  }
+  const json = await response.json();
+  console.log(path, json);
```

Comment:
Was this intentional, or left over from testing?

Reply:
This is intentional--see the test plan. We'll remove it once we actually have a frontend that displays data.